src/tools/genpolicy/src/utils.rs

```diff
@@ -82,13 +82,23 @@ struct CommandLineOptions {
         help = "Registry that uses plain HTTP. Can be passed more than once to configure multiple insecure registries."
     )]
     insecure_registry: Vec<String>,
+
+    #[clap(
+        short,
+        long,
+        help = "If specified, resources that have a runtimeClassName field defined will only receive a policy if the parameter is a prefix of the name",
+        default_missing_value = "kata-cc", // used if flag is present but no value is given
```

Comment:
Please remove this default value - if I understand correctly, that's not a commonly name used in Kata CI, etc.